crates/masonry/src/event_loop_runner.rs

```diff
@@ -307,22 +307,35 @@ impl MainState<'_> {
 pub(crate) fn try_init_tracing() -> Result<(), SetGlobalDefaultError> {
     #[cfg(not(target_arch = "wasm32"))]
     {
+        use time::macros::format_description;
         use tracing_subscriber::filter::LevelFilter;
+        use tracing_subscriber::fmt::time::UtcTime;
         use tracing_subscriber::prelude::*;
         use tracing_subscriber::EnvFilter;
 
+        // Default level is DEBUG in --dev, INFO in --release
+        // DEBUG should print a few logs per low-density event.
+        // INFO should only print logs for noteworthy things.
         let default_level = if cfg!(debug_assertions) {
             LevelFilter::DEBUG
         } else {
             LevelFilter::INFO
         };
+        // Use EnvFilter to allow the user to override the log level without recompiling.
+        // TODO - Print error message if the env var is incorrectly formatted.
         let env_filter = EnvFilter::builder()
             .with_default_directive(default_level.into())
             .with_env_var("RUST_LOG")
             .from_env_lossy();
+        // This format is more concise than even the 'Compact' default:
+        // - We print the time without the date (GUI apps usually run for very short periods).
+        // - We print the time with seconds precision (we really don't need microseconds=.
+        // - We don't print the target
```

Comment:
Not sure about this either - it seems useful to have some context about where a log is coming from?

Also, this comment doesn't seem to serve any purpose - it only duplicates the `with_target` line. It should probably explain why we exclude the target

Reply:
My experience with these logs so far is that the target is visual noise. It almost never gives me info I'm looking for.

If you have a log message, you can already grep the log's location. The target only gives you "Which module was this log defined in?", which I've never found useful.

You're right that I should include this explanation.